docs/integrations/app-integrations/youtube.mdx

```diff
@@ -5,66 +5,79 @@ sidebarTitle: YouTube
 
 In this section, we present how to connect YouTube to MindsDB.
 
-[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users can upload, view, share, and interact with videos created by individuals and organizations from around the world.
+[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users
+can upload, view, share, and interact with videos created by individuals and organizations from around the world.
 
-Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments, detect sentiment of comments, or use AI models to create replies.
+Comment Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments,
+detect sentiment of comments, or use AI models to create replies.
 
-## Connection
+## Install the Handler
+
+First, we have to install the required python packages.
+```bash
+pip install mindsdb[YouTube]
+```
+
+## Connecting
 
 This handler is implemented using the `google-api-python-client` library, a Python library that from where we can use YouTube v3 API.
 
 The only required argument to establish a connection are as follows:
 
-* `youtube_api_token` which is a Google API key used for authentication.
+* `YouTube_api_token` which is a Google API key used for authentication.
 
 <Tip>
-Check out [this guide](https://blog.hubspot.com/website/how-to-get-youtube-api-key) on how to create the API key in order to access YouTube data.
+Check out [this guide](https://blog.hubspot.com/website/how-to-get-YouTube-api-key) on how to create the API key in order to access YouTube data.
 </Tip>
 
 The first step is to create a database with the new `Youtube` engine using the following command:
 
 ```sql
 CREATE DATABASE mindsdb_youtube
-WITH ENGINE = 'youtube',
+WITH ENGINE = 'YouTube',
 PARAMETERS = {
-  "youtube_api_token": "<your-youtube-api-key-token>"  
+  "YouTube_api_token": "<your-YouTube-api-key-token>"  
 };
 ```
 
 <Tip>
-If you installed MindsDB locally via pip, you need to install all handler dependencies manually. To do so, go to the handler's folder (mindsdb/integrations/handlers/youtube_handler) and run this command: `pip install -r requirements.txt`.
+If you installed MindsDB locally via pip, you need to install all handler dependencies manually. To do so, go to the
+    handler's folder `mindsdb/integrations/handlers/YouTube_handler` and run this command: `pip install -r requirements.txt`.
 </Tip>
 
 ## Usage
 
-Use the established connection to query the `get_comments` table for any YouTube Video of your choice.
-
-```sql
-SELECT * FROM mindsdb_youtube.get_comments
-WHERE youtube_video_id = "raWFGQ20OfA";
-```
+Use the established connection to query the comments table
 
+~~~~sql
```

Comment:
In the MDX format, we use triple backticks, so please change it back throughout this doc page.